job/replay_manager.go

```diff
@@ -37,7 +40,9 @@ type ReplayManagerConfig struct {
 
 type ReplayManager interface {
 	Init()
-	Replay(context.Context, *models.ReplayWorkerRequest) (string, error)
+	Replay(context.Context, *models.ReplayRequest) (string, error)
+	GetReplay(uuid uuid.UUID) (*models.ReplaySpec, error)
```

Comment:
Any reason we are using pointer here as well?

Reply:
Have updated this too to not use pointer